authentication/index.adoc

```diff
@@ -0,0 +1,65 @@
+Authorization involves determining whether a user has the required permissions to perform an action.
+
+Based on how users interact with an {product-title} cluster, users can be of xref:../authentication/understanding-authentication.adoc#rbac-users_understanding-authentication[three types] - regular users, system users, and service accounts. By default, each user type has a set of access permissions and privileges. These permissions and privileges are represented through the xref:../authentication/using-rbac.adoc#authorization-overview_using-rbac[role-based access control (RBAC) objects], such as rules, roles, and bindings.
+
+Administrators can manage authorization using the RBAC objects. You can also control access to an {product-title} cluster through xref:../authentication/using-rbac.adoc#rbac-projects-namespaces_using-rbac[projects and namespaces].
+
+An administrator can configure authorization for {product-title} through the following tasks:
+
+* xref:../authentication/using-rbac.adoc#creating-cluster-admin_using-rbac[Create a cluster-admin user]: By default, your cluster has only one cluster administrator called `kubeadmin`. You can create another cluster administrator after defining an identity provider.
++
+[NOTE]
+====
+After creating the cluster admin user, xref:../authentication/remove-kubeadmin.adoc#removing-kubeadmin_removing-kubeadmin[delete the existing kubeadmin user] to improve cluster security.
+====
+* xref:../authentication/understanding-and-creating-service-accounts.adoc#service-accounts-managing_understanding-service-accounts[Create service accounts]: xref:../authentication/understanding-and-creating-service-accounts.adoc#service-accounts-overview_understanding-service-accounts[Service accounts] provide a flexible way to control API access without sharing a regular user’s credentials. A user can xref:../authentication/using-service-accounts-in-applications.adoc#service-accounts-using-credentials-externally_using-service-accounts[use a service account in applications] and also as xref:../authentication/using-service-accounts-as-oauth-client.adoc#using-service-accounts-as-oauth-client[an OAuth client]   .
+* xref:../authentication/tokens-scoping.adoc#tokens-scoping[Scope tokens]: A scoped token is a token that identifies as a specific user who can perform only specific operations. You can create scoped tokens to delegate some of your permissions to another user or a service account.
+* xref:../authentication/managing-security-context-constraints.adoc#managing-pod-security-policies[Control permissions for pods using security context constraints (SCCs)]: You can use SCCs to define a set of conditions that determine the actions a pod can perform and the resources it can access.
+* Impersonate a system admin user: You can grant cluster administrator permissions to a user by xref:../authentication/impersonating-system-admin.adoc#impersonating-system-admin[impersonating  system admin user].
```

Comment:
You need an "a" or "the" in "impersonating  system admin user"

Also, I'm not sure if impersonating the system:admin user is the same thing as "grant cluster administrator permissions to a user", so I'd check with an SME on that.